### DISCUSSION TITLE
rtx shell completion weird question

### DISCUSSION THREAD

**Discussion by tony-sol** [jdx/mise#1092] [Q&A] [answered]
After update, my shell completion for rtx just stops working.

My `.zshrc` setup is
```bash
eval "$(rtx activate $(basename -- ${SHELL}))"
source <(rtx completion zsh)
```

but on tab press i get
```bash
rtx _arguments:465: command not found: __rtx_cmds
```

I've tried to look at compdef logs, but can't find any helpful (on attach)
[zsh57539rtx1.txt](https://github.com/jdx/rtx/files/13588939/zsh57539rtx1.txt)

Is there something i missed up?

**Comment by Pilgrim1379** [comment]
Yes same here and I'm not sure why?

**Reply by jdx** [reply, marked as answer]
should be resolved in the latest version:

```
❯ rtx -v
2023.12.19 macos-arm64 (368ae72 2023-12-07)

~
❯ source <(rtx completion zsh)


~
❯ rtx install 10.0.0
bun         erlang      java        python      shellcheck  zig
deno        go          jq          raku        shfmt       zigmod
elixir      grain       node        ruby        tiny
```
